### DISCUSSION TITLE
Ventura-Monterey-OC-0.9.5-Release-Infinix-InBook-Series

### DISCUSSION THREAD

**Discussion by devboloji** [devboloji/Infinix-Hackintosh-Opencore-Guide#35] [Discuss about the release]
- Moved to Opencore 0.9.5 stable. 
- Changed 1.5GB Vram from 4GB vRam. For better graphics optimization & UI smoothness for the device.
- Added native USB tethering Support so you can use your android as ethernet device. If native tethering is not working for you, just install HoRNDIS [here](https://github.com/jwise/HoRNDIS/releases)
- Updated all the required kexts except trackpad which makes bootloop if we use latest. 
- Generate Your SMBIOS if you need iservices like imessages, etc.
- Great stability if you are using Monterey.
- Now the min temperature of the device is 40 degrees in Monterey(You may face some issues with temparature in Ventura). Temparatures are better than Windows and Linux in MacOS BigSur or Monterey(not Ventura).
- Fixed Bluetooth. From now onwards you won't see "Volume HashMismatch" in Monterey. Bluetooth is 100% functional for X1 i3 Monterey,but not in Ventura. You see bluetooth issues and app closing issues in Ventura due to it's new. If you face any issue with Bluetooth or malfunction, just Open a terminal and type `sudo purge` . If it doesn't work, just reboot. Keep on using Monterey, you don't face bluetooth issue if you use your device for long days like 15days or a month with this OS.
- This will be the last update if there is no contribution or donations from you.
### **Known issues**:(but no one reported):-
- Right side,One usb 3.0 is ded if we you plug in a USB 3.0 in X1.
- Don't worry about Camera in InBook X2 or slim, Update will be released, but I need a X2 & slim user using MACOS. contact me : [MyTelegram](https://t.me/Pappusaidev). Just report me here or Raise an issue. Telegram PM also works.
- Make a list of issues which are not working or need to be fixed in x2, slim or further version of Infinix InBook and bring it to my eye. So I can fix those. Because I have only X1.
- I didn't test this Ventura EFI, If it worked let me know in this GitHub discussion.

- **If you enjoyed this project — or just feeling generous, consider buying me a coffee. Cheers! ☕ :**
# :diamond_shape_with_a_dot_inside:buy me a coffee :coffee:  Donate :  :diamond_shape_with_a_dot_inside:
donate via paypal :
<a href="https://www.paypal.me/369639/"><img src="blue.svg" height="40"></a>  
donate via Paytm:
| <a><img src="darkblue.svg" height="40"></a> |
|-------------------------------------------|
|  `7095160636797@paytm` |

<hr /><em>This discussion was created from the release <a href='https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide/releases/tag/infinixInBook-OC-0.9.5'>Ventura-Monterey-OC-0.9.5-Release-Infinix-InBook-Series   </a>.</em>

**Comment by barrrliebert** [comment]
help me stuck in boot picker dont show macos instaler

**Reply by devboloji** [reply]
If you cannot see picker, Use EFI 8.5 of mine just for installation.

**Reply by barrrliebert** [reply]
how to fix audio doesn't detect?

**Reply by devboloji** [reply]
@barrrliebert before you ask a question or fix, mention your device specifications.

**Reply by barrrliebert** [reply]
infinix inboox x1 pro

**Reply by devboloji** [reply]
@barrrliebert change alcid to 99 then reboot
